<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TInvestor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="stocksPage">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="stocksListWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="simulationPage"/>
      <widget class="QWidget" name="autoPilotPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefreshManually"/>
   <addaction name="separator"/>
   <addaction name="actionStocksPage"/>
   <addaction name="actionSimulationPage"/>
   <addaction name="actionAutoPilotPage"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionRefreshManually">
   <property name="icon">
    <iconset resource="../../Resources.qrc">
     <normaloff>:/assets/images/refresh.png</normaloff>:/assets/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh manually (F5)</string>
   </property>
   <property name="toolTip">
    <string>Manually refresh stock information (F5)</string>
   </property>
   <property name="shortcut">
    <string notr="true">F5</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionStocksPage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Resources.qrc">
     <normaloff>:/assets/images/stocks.png</normaloff>:/assets/images/stocks.png</iconset>
   </property>
   <property name="text">
    <string>Stocks (Ctrl+1)</string>
   </property>
   <property name="toolTip">
    <string>Show page with list of stocks (Ctrl+1)</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+1</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSimulationPage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Resources.qrc">
     <normaloff>:/assets/images/simulation.png</normaloff>:/assets/images/simulation.png</iconset>
   </property>
   <property name="text">
    <string>Simulation (Ctrl+2)</string>
   </property>
   <property name="toolTip">
    <string>Show page with simulation mode (Ctrl+2)</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+2</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAutoPilotPage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Resources.qrc">
     <normaloff>:/assets/images/auto_pilot.png</normaloff>:/assets/images/auto_pilot.png</iconset>
   </property>
   <property name="text">
    <string>Auto-pilot (Ctrl+3)</string>
   </property>
   <property name="toolTip">
    <string>Show page with auto-pilot mode (Ctrl+3)</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+3</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../Resources.qrc">
     <normaloff>:/assets/images/settings.png</normaloff>:/assets/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings (F6)</string>
   </property>
   <property name="toolTip">
    <string>Settings (F6)</string>
   </property>
   <property name="shortcut">
    <string notr="true">F6</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
